src/cobalt.rs

```diff
@@ -7,6 +7,7 @@ use liquid::Value;
 use rss;
 use jsonfeed::Feed;
 use jsonfeed;
+use sass_rs::{compile_file as sass_compile_file, Options as SassOptions};
```

Comment:
If you're just going to rename them to effectively include the crate name in the names, why bother doing anything more than `use sass_rs;`

(Granted I'm biased by my general dislike of this style of `user` despite it being effectively a standard practice)